Added ROVTIO

aayush-fadia opened the pull request:
Added ROVTIO, configs, dependencies and install instructions.

aayush-fadia committed:
Added ROVTIO.
+ dependencies
+ instructions in README.md

aayush-fadia commented:
Will cleanup configs if necessary.

p-amyjiang commented:
What's changes between this and the version of ROVTIO that was demo'ed last week?

p-amyjiang commented on README.md:
````diff
@@ -1 +1,32 @@
+```
+
+### Important
+Touch grass.
````
Does this make ROVTIO build faster

aayush-fadia replied:
Yes, is important.
ROVTIO does perform better outdoors.

p-amyjiang commented:
> Will cleanup configs if necessary.

Yes
Also add run command (and appropriate config) to readme

aayush-fadia committed:
Added good configs (from PC).

Misc. File changes.
Changed paths to configs and odometry out topic to match MAVROS.

aayush-fadia committed:
Changed README, pointed to correct launch file, build command modified to build with 8 max features and 2 cameras.

aayush-fadia committed:
Fixed paths.

aayush-fadia committed:
Viz off, path fix, frame fix for MAVROS and PixHawk.

aayush-fadia commented:
Cleaned up config files, have a configuration that is ready to run. Some tuning directions:
1. Time delta between cam image and IMU image.
2. Use Scaife Hall drone cage MoCap to collect GT drone trajectory data, tune using wandb.

p-amyjiang commented:
LGTM
